apps/wallet/src/ui/app/redux/slices/account/index.ts

```diff
@@ -143,4 +140,9 @@ const reducer: Reducer<typeof initialState> = accountSlice.reducer;
 export default reducer;
 
 export const activeAccountSelector = ({ account }: RootState) =>
+    Object.values(account.entities).find(
```

Comment:
we could use `accountsAdapterSelectors.selectById` here to avoid accessing the state that the adapter creates. (We already set the address as id for the entities on line 65)